pybossa/repositories/project_repository.py

```diff
@@ -180,6 +180,22 @@ def _verify_data_classification(self, project):
         if data_access:
             project.info['data_access'] = [data_access]
 
+    def _verify_required_fields(self, project):
+        if not project.name:
+            raise BadRequest("Name required")
+        if not project.short_name:
+            raise BadRequest("Short_name required")
+        if not project.description:
+            raise BadRequest("Description required")
+        kpi = project.info.get("kpi")
+        if kpi is None:
+            raise BadRequest("KPI required")
+        if not isinstance(kpi, (float, int)) or kpi > 120 or kpi < 0.1:
+            raise BadRequest("KPI must be value between 0.1 and 120")
+        if not project.info.get("product"):
```

Comment:
Could we also check if the value for product and subproduct exists in the dictionary config value?